Assets/Plugins/Source/Editor/Internal/EOSPackageExportConfiguration.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Playeveryware.Editor
+{
+    [Serializable]
+    public class PlatformExportInfo
+    {
+        [SerializeField]
+        public string platform;
+
+        [SerializeField]
+        public string descPath;
+
+        [SerializeField]
+        public bool isGettingExported;
+    }
+
+    [Serializable]
+    public class PlatformExportInfoList
+    {
+        [SerializeField]
+        public List<PlatformExportInfo> platformExportInfoList;
+
+    }
+
+    [Serializable]
+    public class SampleExportInfo
+    {
+        [SerializeField]
+        public string sample;
+
+        [SerializeField]
+        public string descPath;
+
+        [SerializeField]
+        public bool isGettingExported;
+    }
```

Comment:
I think these definitions can be made into nested classes in places where they are utilized. This helps us avoid the pattern of having more than one definition per file, and also side-steps the issue of having a filename that doesn't properly represent the contents it contains.